llama-index-core/llama_index/core/base/embeddings/base.py

```diff
@@ -347,6 +352,7 @@ async def aget_text_embedding_batch(
     ) -> List[Embedding]:
         """Asynchronously get a list of text embeddings, with batching."""
         dispatch_event = dispatcher.get_dispatch_event()
+        num_workers = self.num_workers
```

Comment:
Should `num_workers` just be a method argument of `aget_text_embedding_batch`?

Reply:
we would need to put on __call__ as well, to work with transformations

Reply:
I think its in line with the current interface (batch size is also a class attribute)